stories/Button/Guideline.mdx

```diff
@@ -0,0 +1,169 @@
+import {
+  Meta,
+  Story,
+  Canvas,
+  Primary,
+  Controls,
+  Unstyled,
+  Source,
+} from '@storybook/blocks';
+import { Button } from '../../src/lib/components/buttonv2/Buttonv2.component';
+
+import * as ButtonStories from './button.stories';
+
+<Meta of={ButtonStories} />
+
+# Button
+
+Buttons are used to trigger an action or event when clicked.
```

Comment:
I would change _clicked_ by "activated" or "triggered". This because you can trigger the action not only by a click but also by a form submission of via a keyboard action on the button

Reply:
I would use "triggered", as "activated" is to close to the word "active", which is a status.